### PR TITLE
Email addresses can't contains quotes as it should be

### LINKED ISSUE #734: Tickets are not created by emails when email addresses contain quotes
Hello,
If an email address contains quotes (yes it's possible, see: [Email Address](https://en.wikipedia.org/wiki/Email_address#Valid_email_addresses)) then the ticket is not created as the user can't be extracted from the DB.
The problem is in `getOrImportByEmail()` function in `user.class.php` file.

I'll push a fix for that.

regards,
Thank you,

Tomolimo

### PR DESCRIPTION
**Pull request by tomolimo**
Fixes #734 

Added possibility to view email addresses for users in user form, and in user list.
Added possibility to create tickets from emails when sender address contains quotes.

regards,

Tomolimo

### CODE REVIEW

File: inc/user.class.php [hunk 1]
```diff
@@ -3657,7 +3657,7 @@ static function getOrImportByEmail($email='') {
       $query = "SELECT `users_id` as id
                 FROM `glpi_useremails`
                 LEFT JOIN `glpi_users` ON (`glpi_users`.`id` = `glpi_useremails`.`users_id`)
-                WHERE `glpi_useremails`.`email` = '".stripslashes($email)."'
+                WHERE `glpi_useremails`.`email` = '".$DB->escape(stripslashes($email))."'
```
Comment (remicollet) [inline inc/user.class.php]: stripslashes not need if you add DB->escape

Comment (tomolimo) [inline inc/user.class.php]: good, then I commit a new version in 1 min

Comment (tomolimo) [inline inc/user.class.php]: But: are you sure stripslashes is going to delete any slashes, and escape will add only slashes where they are needed

Comment (remicollet) [inline inc/user.class.php]: forget my previous comment, I was thinking you add replace addslashes by escape...

**Pull request merged**